src/components/Project/ProjectBasics/ProjectForm/ProjectFormBanner.tsx

```diff
@@ -9,13 +12,40 @@ interface IProjectFormbannerProps {
 }
 const ProjectFormBanner: FC<IProjectFormbannerProps> = ({ onSubmit, isDirty }) => {
   const { t } = useTranslation();
+  const project = useAppSelector(selectProject);
+  const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false);
 
+  const handleDialogClose = useCallback(() => {
+    setIsDialogOpen(false);
+  }, []);
+  const handleDialogOpen = useCallback(() => {
```

Comment:
We should replace this with the useConfirmDialog-hook. The ProjectDeleteDialog can also be removed once done.

Reply:
Something like this:

 const { isConfirmed } = useConfirmDialog();  

 const handleDeleteProject = useCallback(async () => {
  try {
    const confirm = await isConfirmed({ title: 'Your dialog title', description: 'Your dialog description' });

  if (confirm !== false) {
    // do delete stuff
  }
  }catch(e){
   // catch stuff
  }
  }, []);